cpp/src/rolling/rolling.cu

```diff
@@ -25,275 +25,21 @@
 #include <cudf/utilities/nvtx_utils.hpp>
 #include <cudf/utilities/bit.hpp>
 #include <cudf/detail/copy.hpp>
+#include <cudf/detail/rolling/rolling.cuh>
 #include <cudf/detail/utilities/cuda.cuh>
 #include <cudf/copying.hpp>
 
 #include <rmm/device_scalar.hpp>
 
+#include <thrust/binary_search.h>
```

Comment:
I understand why code was removed from this file, but why were new includes added?

Reply:
These are unnecessary remnants of the refactor. I should've removed them. Will do.